GG_clone_backend/db/seeds.rb

```diff
@@ -5,25 +5,63 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-Passenger.create({ first_name: 'Hakob', last_name: 'Arshakyan', phone_number: '+37444556699',
-                   email: 'harshakayan@mail.com', password_digest: Passenger.digest('Passenger1') })
-Passenger.create({ first_name: 'Levon', last_name: 'Hakobyan', phone_number: '+37444121232',
-                   email: 'lakobyan777thug@mail.com', password_digest: Passenger.digest('Passenger2') })
-Passenger.create({ first_name: 'Gegham', last_name: 'Antonyan', phone_number: '+37455656522',
-                   email: 'gexamo@mail.com', password_digest: Passenger.digest('Passenger3') })
+# Passenger.create({ first_name: 'Hakob', last_name: 'Arshakyan', phone_number: '+37444556699',
```

Comment:
if we're going to deprecate this part talk with @iwazowsky and if he's ok with faker approach remove commented section

Reply:
I need this seeds for testing purposes,to achieve predictable results, as I use exact same credentials in api and frontend automation, I guess faker approach will not give me predictable data, am I right?